CHANGELOG.md

```diff
@@ -29,6 +30,47 @@
 - [v0.2.0](#v020---20190401)
 - [v0.1.0](#v010---20190112)
 
+## [v1.7.0] - 2021/05/20
+
+### Added
+
+- State files now support environment variable-based templating. decK can
+  substitute the value of an environment variable into an object in the state
+  file. This is useful for avoiding persistent cleartext storage of sensitive
+  values and populating values that vary between similar configurations.
+  [#286](https://github.com/Kong/deck/pull/286)
+- Sort state file objects by name, to ease comparing state files from
+  similarly-configured instances that do not share object IDs.
+  [#327](https://github.com/Kong/deck/pull/327)
+- Added a default timeout to HTTP requests.
+- Implemented `convert` command for converting state files between Kong Gateway
+  and Konnect configuration formats.
+  [#330](https://github.com/Kong/deck/pull/330)
+- Add `--konnect-addr` flag to set Konnect address.
+  [#374](https://github.com/Kong/deck/pull/374)
+
+### Fixed
+
+- Fixed duplicate error message prints.
+  [#317](https://github.com/Kong/deck/pull/317)
+- Handle mtls-auth credential API behavior when Kong Enterprise is running in
+  free mode. decK no longer treats the free mode mtls-auth behavior as a fatal
+  error.
+  [#321](https://github.com/Kong/deck/pull/321)
+- `--select-tag` tags are now applied to credentials.
+  [#282](https://github.com/Kong/deck/pull/282)
+- Fix empty Service Package descriptions not syncing correctly.
+  [#347](https://github.com/Kong/deck/pull/347)
+- Updating certificate fields no longer deletes SNI associations. 
+  [#386](https://github.com/Kong/deck/pull/386)
```

Comment:
19205e5

Reply:
Wrong commit link? This is #347, which is above.

Reply:
My bad. I thought that commit/PR was not documented.